doh-server/config.go

```diff
@@ -42,6 +42,8 @@ type config struct {
 	Verbose          bool     `toml:"verbose"`
 	DebugHTTPHeaders []string `toml:"debug_http_headers"`
 	LogGuessedIP     bool     `toml:"log_guessed_client_ip"`
+	LocalIPFilter    bool     `toml:"local_ip_filtering"`
+	ECSFullSubnet    bool     `toml:"ecs_full_subnet"`
```

Comment:
As I said in the issue, if you want to reorder the options, remember to reorder the variables here also.